tests/vinyldns_tests.bats

```diff
@@ -166,6 +159,21 @@ load test_helper
   [ "${output}" = "${fixture}" ]
 }
 
+@test "search-record-sets" {
+  run $ew search-record-sets \
+    --record-name-filter "so*" \
+    --record-type-filter "CNAME" \
+    --record-type-filter "mx" \
+    --max-items "50" \
+    --name-sort "DESC"
+  record_sets[0]="some-cname"
+  record_sets[1]="some-mx"
+  for record_set in ${record_sets}
+  do
+    [[ "${output}" =~ "${record_set}" ]]
+  done
```

Comment:
This looks reasonable, though I'm curious to hear a bit more about why you chose to diverge from the established pattern in the other tests where a "fixture" is used.

Reply:
At the time of writing this test case, I went with a simple way to test. Now the test case is updated to match the established pattern.